### ISSUE TITLE
Crash on project viewer on windows

### ISSUE THREAD

**Issue by robmcmullen** [robmcmullen/peppy#740]
_[Trac time 20090807 210333Z]_ From an emailed bug report:
1. I started Peppy.
1. I loaded peppy.py (I downloaded the whole source today)
1. Click on Projects (left sidebar)
1. Doble-click on .peppy-project
1. Crash

Hope this helps!
Traceback:

```
#---- System Information ----#
Peppy Version: 0.10.0
Operating System: Windows XP (build 2600, Service Pack 3)
Python Version: 2.5.4 (9e410e838757228470a098afd8777aed4a122327:67916, Dec 23 2008, 15:10:54) [MSC v.1310 32 bit (Intel)]
wxPython Version: 2.8.9.2 (msw-unicode)
wxPython Info: (__WXMSW__, wxMSW, unicode, wx-assertions-on, SWIG-1.3.29)
Python Encoding: Default=ascii  File=mbcs
wxPython Encoding: latin1
System Architecture: 32bit 
Byte order: little
Frozen: windows_exe
#---- End System Information ----#

#---- Traceback Info ----#
*** Fri Aug 07 13:55:14 2009 ***
Traceback (most recent call last):
  File "peppy\project\projecttree.pyo", line 165, in OnCompareItems
TypeError: 'NoneType' object is unsubscriptable

#---- End Traceback Info ----#
```

**Comment by robmcmullen** [comment]
_[Trac time 20090807 210944Z]_ Appears to happen when double clicking on any directory in the project list, but only in the py2exe version.  It's fine when running peppy from a source distribution.  Also, using the expand button expands the folder without issue -- it's related to the double click action only, apparently.

**Comment by robmcmullen** [comment]
_[Trac time 20090809 215348Z]_ 6516e31f7301e6a32445d841777ac1749399b3e7 Fixed #740: pulled fixes for MyTreeCtrl and ProjectTree.OnCompareItems from upstream Editra project
- double click wasn't expanding node on MSW until upgrading to wxPython 2.8.10.1
